src/libraries/System.Data.Common/src/System/Data/Common/DbConnectionStringBuilder.cs

```diff
@@ -587,7 +587,7 @@ private PropertyDescriptorCollection GetProperties(Attribute[]? attributes)
         {
             // Below call is necessary to tell the trimmer that it should mark derived types appropriately.
             // We cannot use overload which takes type because the result might differ if derived class implements ICustomTypeDescriptor.
-            Type thisType = GetType();
+            Type _ = GetType();
```

Comment:
How are you deciding when to use `_` and when to just ignore the return value?
```suggestion
            GetType();
```

Reply:
I was not sure if linker analysis will be able to recognize GetType () + pop in the same way

Reply:
Deleting unused local variables like this is just a style change. I do not think it affects the IL. Roslyn compiles assignments of results into unused variables as `pop`, without actually creating the local variable for the result.

For example: https://sharplab.io/#v2:C4LglgNgPgAgTARgLAChUBUCeAHApgAmHwF58A7XAd3wHsAjAK1wGNgAKASgDoBxXYLHk4BuIA==

Reply:
ok, good to know (will update this)